tests/input/null_coalesce_operator.php

```diff
@@ -4,8 +4,6 @@
 
 $foo = isset($_GET['foo']) ? $_GET['foo'] : 'foo';
 
-$bar = isset($bar) ? $bar : 'bar';
-
```

Comment:
Moved to `tests/input/null_coalesce_equal_operator.php` as the fixed output now contains an `??=`.